ply/ply.c

```diff
@@ -1431,7 +1429,7 @@ void ascii_get_element(PlyFile *plyfile, char *elem_ptr)
   char **words;
   int nwords;
   int which_word;
-  char *elem_data,*item;
+  char *elem_data,*item=NULL;
```

Comment:
These should perhaps use constructors instead of assignments? e.g. char *item(NULL);